### ISSUE TITLE
Add an option to tab preferences to switch previews on/off

### ISSUE THREAD

**Issue by wolfbeast** [MoonchildProductions/Pale-Moon#1444]
Since it remains a common question for people, we should expose `browser.ctrlTab.previews` in the UI.

This is a good first issue for new contributors.

**Comment by stilez** [comment]
Also **browser.ctrlTab.mostRecentlyUsed** - the choice of next/prev or most recent, for tab switching, is one where each person may has a strong preference. Well worth exposing.

**Comment by wolfbeast** [comment]
The MRU only applies to using the graphical tab switcher. Because it only lists a (very) limited number of tabs, does it make sense to **not** use the MRU list there? Is that a common workflow for people?

**Comment by graciecat** [comment]
::confusion emoji:: There is no **browser.ctrlTab.mostRecentlyUsed**. The preference  **browser.ctrlTab.mostRecentlyUsedLimit** is what @stilez meant, right?  But that doesn't control the choice of next/prev vs. most recent; that's controlled by **browser.ctrlTab.previews** . When you turn previews off (set to _false_), then Ctrl-Tab selects next tab and Ctrl-Shift-Tab selects previous tab. When you turn previews on (set to _true_), then Ctrl-Tab selects the most recent tab (so you end up toggling between two tabs) and Ctrl-Shift-Tab brings up the tabs preview.

To answer your question @wolfbeast, IMHO people who use the graphical tab switcher probably don't have a need to change the number shown that frequently. If they can't find it in 7 tabs, they click on "Show all _n_ tabs" (at least that's my workflow, anyhow). 

Or are you asking about removing this feature completely? I do use the MRU list, so I wouldn't want to see it removed.

**Comment by wolfbeast** [comment]
@graciecat I was asking about exposing the use of the MRU list in the UI, because I don't find that necessary. Neither do you, looks like, and I also can't see any common-used workflow _with_ previews where people would not want to use the MRU list in the tab switching UI.

**Comment by graciecat** [comment]
@wolfbeast, I recently learned that whether or not browser.ctrlTab.previews is set to _true_ or _false_, there are already keyboard shortcuts that take you to the previous/next tab, which is what @stilez and other users at the PM forumcx are looking for. Ctrl+PageUp selects the previous tab and and Ctrl+PageDown selects the previous tab.

It still may be worth it to proceed with this issue because people coming from FF or SM might expect Ctrl+Tab and Ctrl+Shift+Tab to behave in a particular way. (I guess that is something to decide.)

Since I just discovered this shortcut by accident, it sort of points to a need as well for an easily accessible list of keyboard shortcuts so that users looking for particular actions/command may find there is an easy alternative to changing configuration settings.
